.github/workflows/os-llvm-sycl-build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build with Open Source LLVM-Sycl compiler
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  install-compiler:
+    name: Install Nightly build of DPC++ toolchain
+    runs-on: ubuntu-20.04
+
+    env:
+      OCLCPUEXP_FN: oclcpuexp-2021.12.6.0.19_rel.tar.gz
```

Comment:
Is there a way to populate these values dynamically? Otherwise, we will soon be using a older driver and not be up-to-date with latest release.

Reply:
intel/llvm uses https://github.com/intel/llvm/blob/sycl/buildbot/dependency.conf